internal/model/modelhelpers/assertions.go

```diff
@@ -13,3 +13,10 @@ func AssertNonEmptyCategories(t *testing.T, categories []model.Category) {
 		assert.NotEmpty(t, category.CategoryType, "unexpected empty category type")
 	}
 }
+
+func AssertNonEmptyOrganizations(t *testing.T, organizations []model.Organization) {
+	for _, organization := range organizations {
+		assert.NotEmpty(t, organization.Id, "unexpected empty organization id")
+		assert.NotEmpty(t, organization.Name, "unexpected empty organization name")
+	}
+}
```

Comment:
Add more fields to check here (e.g created_by, create_date etc etc)

Reply:
The fields have now been added.